intercom_flutter/example/android/app/src/main/kotlin/io/maido/intercomexample/MainActivity.kt

```diff
@@ -1,6 +0,0 @@
-package io.maido.intercomexample
-
-import io.flutter.embedding.android.FlutterActivity
-
-class MainActivity() : FlutterActivity() {
-}
```

Comment:
We should keep this file and extend it with `FlutterFragmentActivity`

Reply:
Why? It's not being used, so I imported it directly in the manifest

Reply:
This will be easy to understand for the users. And thats what is mentioned in docs.